plugins/Kubernetes/v1/docs/README.md

```
@@ -0,0 +1,136 @@
+        apiGroup: rbac.authorization.k8s.io
+        kind: ClusterRole
+        name: squaredup-reader
+    subjects:
+        - kind: ServiceAccount
+          name: squaredup-reader
+          namespace: kube-system
+    ```
+
+2. Generate a long-lived token for the ServiceAccount:
+
+    ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)